resources/dicts/patrols/general/training.json

```diff
@@ -11690,5 +11690,371 @@
                 "weight": 20
             }
         ]
-    }
+    },
+    {
+       "patrol_id": "gen_train_advanced_moves",
+       "biome": ["any"],
+       "season": ["any"],
+       "types": ["training"],
+       "tags": [],
+       "patrol_art": "train_general_intro",
+       "min_cats": 2,
+       "max_cats": 2,
+       "min_max_status": {
+           "normal adult": [1, 6],
+           "all apprentices": [1, 6]
+       },
+   
+       "weight": 20,
+       "chance_of_success": 50,
+       "intro_text": "As p_l leads app1 to a training area, app1 stops {PRONOUN/p_l/object} and asks, with a serious look in {PRONOUN/app1/poss} eye, if p_l will teach {PRONOUN/app1/object} some <i>advanced</i> battle moves.",
+       "decline_text": "p_l brushes it off and tells app1 {PRONOUN/p_l/subject} {VERB/p_l/have/has} a different plan in mind for today.",
+       "success_outcomes": [
+           {
+               "text": "s_c explains that one day, c_n might be taken over by a tyrannical ruler! True and loyal c_n warriors will have to be powerful enough to keep c_n safe. p_l supposes that's possible and agrees to show {PRONOUN/s_c/object} some advanced moves.",
+               "exp": 10,
+                "stat_trait": ["troublesome", "rebellious", "cunning", "careful"],
```

Comment:
could we add "childish" here? i feel like this can definitely be read in a "overzealous and hyperbolic teen" tone